src/common/components/search-input/search.css

```diff
@@ -0,0 +1,14 @@
+$border-color: $mid-grey;
+
+.searchInput {
+  background-image: url('https://assets.ubuntu.com/v1/4a72426e-search-icon-grey.svg');
+  background-position: right 4px bottom 2px;
```

Comment:
Not that it looks bad, but positioning of search icon in bottom corner seems a little bit off:
<img width="1045" alt="screen shot 2017-07-06 at 16 24 44" src="https://user-images.githubusercontent.com/83575/27915840-adcc59ce-6267-11e7-973e-7f980d136365.png">

And designs have it centered: 

<img width="1045" src="https://cloud.githubusercontent.com/assets/17748020/26348782/b75c4aa6-3fa5-11e7-8843-ca713fd0d1ae.png">